src/scripts/index.js

```diff
@@ -1,3 +1,4 @@
 import '../styles/index.scss';
+import '../index.html';
```

Comment:
Currently the HTML file gets included as a string in the production build. By requiring the file only in development mode, we can avoid this.
```suggestion
if (process.env.NODE_ENV === 'development') {
    require('../index.html')
}
```